lib/bloc_record/selection.rb

```diff
@@ -1,6 +1,12 @@
 require 'sqlite3'
 
 module Selection
+  def method_missing(m, *args, &block)
```

Comment:
The instructions are not super clear here, but your implementation appears to be correct (at least for the one case you have, `find_by_name`.) Sorry for any confusion.